src/content/docs/en/concepts/why-astro.mdx

```diff
@@ -1,45 +1,59 @@
 ---
 title: Why Astro?
-description: "Astro is an all-in-one web framework for building fast, content-focused websites. Learn more."
+description: "Astro is an all-in-one web framework for building fast, content-driven websites. Learn more."
 i18nReady: true
 ---
 
-Astro is an **all-in-one** **web framework** for building **fast,** **content-focused websites.**
+**Astro** is the web framework for building **content-driven websites** like porfolios, blogs, docs, marketing sites, landing pages, and ecommerce. 
 
-Why choose Astro over another web framework? Here are five core design principles to help explain why we built Astro, the problems that it exists to solve, and why Astro may be the best choice for your project or team.
+Astro is best known for pioneering a JavaScript frontend architecture using [islands](/en/concepts/islands), optimized for performance, flexibility, and simplicity.
 
-#### Astro is...
+## Features
 
-1. [Content-focused](#content-focused): Astro was designed for content-rich websites.
+Astro includes everything you need built-in to get started quickly, and hundreds of integrations to extend your site. 
+
+Some highlights include:
+
+- **[Islands:](/en/concepts/islands/)** A component-based web architecture optimized for content-driven websites.
+- **[UI-agnostic](/en/core-concepts/framework-components/):** Supports React, Preact, Svelte, Vue, Solid, Lit, HTMX, web components, and more.
+- **[Server-first:](/en/core-concepts/rendering-modes/)** Moves expensive rendering off of your visitors' devices.
+- **[Zero JS, by default:](/en/core-concepts/astro-components/)** Less client-side JavaScript to slow your site down.
+- **[Content-obsessed:](/en/guides/content-collections/)** Content Collections to validate your Markdown with type-safe frontmatter.
```

Comment:
Perhaps 'focused' or 'immersed' would be more suitable than 'obsessed', as they convey a similar idea but with a more positive tone.

```suggestion
- **[Content-immersed:](/en/guides/content-collections/)** Content Collections to validate your Markdown with type-safe frontmatter.
```

Reply:
Changed this to just "Content Collections", like Islands!